### ISSUE TITLE
import error: cannot import name 'until' from 'theano.scan_module' 

### ISSUE THREAD

**Issue by bfredl** [pymc-devs/pymc#4225]
If you have questions about a specific use case, or you are not sure whether this is a bug or not, please post it to our discourse channel: https://discourse.pymc.io  

## Description of your problem

**Please provide a minimal, self-contained, and reproducible example.**
```python
import pymc3
```

**Please provide the full traceback.**
```python
  File "/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/home/bjorn/.local/lib/python3.8/site-packages/pymc3/__init__.py", line 42, in <module>
    from .distributions import *
  File "/home/bjorn/.local/lib/python3.8/site-packages/pymc3/distributions/__init__.py", line 15, in <module>
    from . import timeseries
  File "/home/bjorn/.local/lib/python3.8/site-packages/pymc3/distributions/timeseries.py", line 22, in <module>
    from .continuous import get_tau_sigma, Normal, Flat
  File "/home/bjorn/.local/lib/python3.8/site-packages/pymc3/distributions/continuous.py", line 27, in <module>
    from .dist_math import (
  File "/home/bjorn/.local/lib/python3.8/site-packages/pymc3/distributions/dist_math.py", line 28, in <module>
    from theano.scan_module import until
ImportError: cannot import name 'until' from 'theano.scan_module' (unknown location)
```

**Please provide any additional information below.**

I searched existing issues for `scan_module` but didn't find anything.

## Versions and main components

* PyMC3 Version: f732a01c6a61caa0fa3dbabc7668ef5ddb5ab286
* Theano Version: 99bb8083f29638e74a47f68a5c5909b9b7874cd1
* Python Version: 3.8
* Operating system: arch linux
* How did you install PyMC3: pip (using git master for both pymc-theano and pymc3)

**Comment by bfredl** [comment]
BTW thanks for you efforts to keep theano alive. It makes my life easier if i can reuse my old theano code. Though some functionality i need (lasagne optimizers etc) are now in pymc3 :sweat_smile:

**Comment by aloctavodia** [comment]
Thanks for the report. Glad you share our excitement about the Theano revival.The issue you are reporting should be addressed by this PR #4218

**Comment by bfredl** [comment]
Thanks for the fix :)
